tests/conftest.py

```diff
@@ -81,3 +82,39 @@ def get_trackers_last_message_none_fixture():
         side_effect=AsyncMock(return_value=response),
     ):
         yield
+
+
+@pytest.fixture(name="get_trackers_phone_call_available_minuend_missing")
+def get_trackers_phone_call_available_minuend_missing_fixture():
+    """Static result when retrieving data from weenect."""
+    response = copy.deepcopy(GET_RACKERS_RESPONSE)
+    response["items"][0]["call_max_threshold"] = None
```

Comment:
```suggestion
    del response["items"][0]["call_max_threshold"]
```

I don't think setting it to `None` makes sense. Rather test for the case that the value is not part of the response at all.